package.json

```diff
@@ -24,24 +24,25 @@
     "url": "git://github.com/rogeriopvl/gulp-jsonlint.git"
   },
   "scripts": {
-    "test": "mocha"
+    "lint": "jshint index.js test/main.js",
```

Comment:
I already have a branch with linting setup with eslint ready to go, so jshint is redundant here.

Reply:
No problem. I'll look at it and try to switch to `eslint`.

Reply:
Sorry, totally forgot to push the code. It's in the master branch now.

Reply:
I merged your changes. However, [Travis fails](https://travis-ci.org/rogeriopvl/gulp-jsonlint/jobs/547097106#L754), because the version of `eslint-plugin-node` that you use does not support Node.js 6. You can either disable `eslint` checks on Node.js 6, or use a different version of the plugin, or not use it.

```txt
/home/travis/build/rogeriopvl/gulp-jsonlint/node_modules/eslint-plugin-node/lib/configs/recommended-module.js:7
        ...commonGlobals,
        ^^^
SyntaxError: Unexpected token ...
```